app/lib/rocketchat.js

```diff
@@ -697,6 +697,12 @@ const RocketChat = {
 		// RC 0.63.0
 		return this.sdk.post('rooms.saveNotification', { roomId, notifications });
 	},
+	saveAudioNotificationValue(rid, params) {
```

Comment:
Missing Rocket.Chat version comment

Reply:
I could not find the version can you tell me https://github.com/RocketChat/Rocket.Chat/blob/d53e264b83ccc971236123853e4b68c9eb6beeb4/app/push-notifications/server/methods/saveNotificationSettings.js#L91

Reply:
never mind I removed it found a way to use over old methord for this task too